aws/websites/squid-proxy/.terraform.lock.hcl

```diff
@@ -0,0 +1,55 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
If this file autogenerated, can you please add it to `.gitattribtues` file to that it'll be excluded from reviews